### PR TITLE
Issue#187 Create JSON Data File

### LINKED ISSUE #187: Create JSON Data Files Using GSOC 2015-2017 data
## Description
As a contributor,
I need create JSON Data file for the years of GSoC 2015-2017,
to automate retrieval of data in future use (In the Programs Page).

### Update [Required]
- New data files

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
4 hours 

@divyanshu-rawat @ancyp @Janiceilene Can you please assign me this issue. :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Added JSON Data files for GSoC 2015-2017

Fixes #187 

### Type of Change:
- Code
  
**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
N/A

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
  

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by BaaniLeen** [comment]
@sara-02 Okay sure! I will try looking up online if I can get that data from somewhere since we do not have the country data for all the members yet. :)

**Comment by sara-02** [comment]
Sure. I will also try to check for some names, and update you with my findings :)

File: src/assets/db/gsoc2016.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+   "PROJECT": "Ushahidi Crowdmap",
+   "STUDENT": "Divya (India)",
+   "PROJECT MANAGER & MENTOR": "Julia (USA)",
+   "MENTORS": ""
+ },
+ {
+   "PROJECT": "Photo Language Translation",
+   "STUDENT": "Rupinderjit (India)",
+   "PROJECT MANAGER & MENTOR": "Rose (USA)",
+   "MENTORS": "Saumya (India); Valerie (USA; Saurabh"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Missed a closing bracket after USA

File: src/assets/db/gsoc2016.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   "PROJECT": "PCSA Web - Automated Testing",
+   "STUDENT": "Anjali (India)",
+   "PROJECT MANAGER & MENTOR": "",
+   "MENTORS": "Mariam"
+ },
+ {
+   "PROJECT": "MACC - Automated Testing",
+   "STUDENT": "Tahirih",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Suryadip (USA; Kesha (India)"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Missed the closing bracket after USA.

File: src/assets/db/gsoc2017.json [hunk 3]
```diff
@@ -0,0 +1,68 @@
+[
+ {
+   "PROJECT": "Photo Language Translation",
+   "STUDENT": "Aditya",
+   "PROJECT MANAGER & MENTOR": "May (USA)",
+   "MENTORS": "Prachi (India); Anjali"
+ },
+ {
+   "PROJECT": "MACC",
+   "STUDENT": "Gunpreet",
```
Comment (sara-02) [inline src/assets/db/gsoc2017.json]: Gunpreet-Canada

File: src/assets/db/gsoc2015.json [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+     "PROJECT":"OVA Android"
+    ,"STUDENT":"Ilo (Nigeria)"
+    ,"PROJECT MANAGER & MENTOR":"Rose (USA)"
+    ,"MENTORS":"Dinu (Sri Lanka)"
+  },
+  {
+     "PROJECT":"PeaceTrack Android"
+    ,"STUDENT":"Bhagya (Sri Lanka)"
+    ,"PROJECT MANAGER & MENTOR":"Nicki (USA)"
+    ,"MENTORS":"Neeraja (USA); Pooja"
```
Comment (sara-02) [inline src/assets/db/gsoc2015.json]: Pooja- India

File: src/assets/db/gsoc2015.json [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+  },
+  {
+     "PROJECT":"PeaceTrack iOS"
+    ,"STUDENT":"Vennela (India)"
+    ,"PROJECT MANAGER & MENTOR":"Nicki (USA)"
+    ,"MENTORS":"Jessica"
```
Comment (sara-02) [inline src/assets/db/gsoc2015.json]: Jessica- USA

File: src/assets/db/gsoc2016.json [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+   "PROJECT": "Malaria (iOS)",
+   "STUDENT": "Teodor (Romania)",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Bruno (Portugal); Jayaruwan"
+ },
+ {
+   "PROJECT": "Malaria (Web)",
+   "STUDENT": "Ananta (India)",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Khushali; Ritesh; Rheeya"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Khushali - USA, Ritesh - USA, Rheeya-USA
All the 3 are currently based out of USA as per their Linkedin, but they are from India.

File: src/assets/db/gsoc2016.json [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+   "PROJECT": "PCSA (iOS)",
+   "STUDENT": "Chamika (Sri Lanka)",
+   "PROJECT MANAGER & MENTOR": "Nicki (USA)",
+   "MENTORS": "Shruti (India)"
+ },
+ {
+   "PROJECT": "PCSA (Web)",
+   "STUDENT": "Akanksha (Sri Lanka)",
+   "PROJECT MANAGER & MENTOR": "Maybellin (USA)",
+   "MENTORS": "Prajakta; Themiya"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Prajakta - USA

File: src/assets/db/gsoc2016.json [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+ },
+ {
+   "PROJECT": "Mobile App Control Center",
+   "STUDENT": "Medha (India)",
+   "PROJECT MANAGER & MENTOR": "",
+   "MENTORS": "Neetu (USA); Irish (Canada); Angel; Lorena"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Angel -USA

File: src/assets/db/gsoc2016.json [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+   "PROJECT": "Mailman 3",
+   "STUDENT": "Aditi (India)",
+   "PROJECT MANAGER & MENTOR": "Ana (Spain)",
+   "MENTORS": "Stephen (Japan)"
+ },
+ {
+   "PROJECT": "PowerUp (Android)",
+   "STUDENT": "Aanchal (India)",
+   "PROJECT MANAGER & MENTOR": "",
+   "MENTORS": "Finda; Chhavi (India); Harshita"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Harshita-USA

File: src/assets/db/gsoc2016.json [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+ },
+ {
+   "PROJECT": "Photo Language Translation - Automated Testing",
+   "STUDENT": "Madi (USA)",
+   "PROJECT MANAGER & MENTOR": "",
+   "MENTORS": "Mariam"
+ },
+ {
+   "PROJECT": "Portal - Automated Testing",
+   "STUDENT": "Vatsala",
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Vatsala- India

File: src/assets/db/gsoc2016.json [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+   "PROJECT": "PowerUp (iOS)",
+   "STUDENT": "Sanya (India)",
+   "PROJECT MANAGER & MENTOR": "",
+   "MENTORS": "Finda; Jennifer (Austria)"
+ },
+ {
+   "PROJECT": "Malaria (Web) - Automated Testing",
+   "STUDENT": "Shraddha",
+   "PROJECT MANAGER & MENTOR": "",
+   "MENTORS": "Shwetambara"
```
Comment (sara-02) [inline src/assets/db/gsoc2016.json]: Shwetambara-India

File: src/assets/db/gsoc2017.json [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+   "PROJECT": "MACC",
+   "STUDENT": "Gunpreet",
+   "PROJECT MANAGER & MENTOR": "Angel",
+   "MENTORS": "Beiyu; Medha"
+ },
+ {
+   "PROJECT": "MACC",
+   "STUDENT": "Yatna (India)",
+   "PROJECT MANAGER & MENTOR": "Angel",
+   "MENTORS": "Beiyu; Medha"
```
Comment (sara-02) [inline src/assets/db/gsoc2017.json]: Medha- India

File: src/assets/db/gsoc2017.json [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+   "PROJECT": "PowerUp iOS",
+   "STUDENT": "Yu",
+   "PROJECT MANAGER & MENTOR": "May (USA)",
+   "MENTORS": "Sally; Ginny; Jennifer"
+ },
+ {
+   "PROJECT": "Mailman 3",
+   "STUDENT": "Aditi (India)",
+   "PROJECT MANAGER & MENTOR": "May (USA)",
+   "MENTORS": "Vatsala; Khushali; Rosario; Stephen"
```
Comment (sara-02) [inline src/assets/db/gsoc2017.json]: Vatsala - India

**Comment by BaaniLeen** [comment]
@sara-02 Thank you soo much for the detailed review! :)
Have added all the countries you suggested as well as a few others I came across from LinkedIn :)

**Comment by ancyp** [comment]
@BaaniLeen it would be good if you could populate the 2009-2015 data if you have time.
Is there an issue for that?

**Comment by ancyp** [comment]
@Janiceilene @sara-02 reviews please, we need 2.

**Comment by Janiceilene** [comment]
I didn't see an issue for 2009-2015. Is that going to be done separately?

**Comment by BaaniLeen** [comment]
Yeah @Janiceilene ! Since that was a stretch goal, I  just created a separate issue for it #192.

**Pull request merged**